miner/miner.go

```diff
@@ -152,6 +154,22 @@ func (miner *Miner) SetGasTip(tip *big.Int) error {
 	return nil
 }
 
+// SetMaxDASize sets the maximum data availability size currently allowed for inclusion. 0 means no maximum.
+func (miner *Miner) SetMaxDASize(maxTxSize, maxBlockSize *big.Int) {
+	miner.confMu.Lock()
+	if maxTxSize == nil || maxTxSize.BitLen() == 0 {
+		miner.config.MaxDATxSize = nil
+	} else {
+		miner.config.MaxDATxSize = new(big.Int).Set(maxTxSize)
+	}
+	if maxBlockSize == nil || maxBlockSize.BitLen() == 0 {
+		miner.config.MaxDABlockSize = nil
+	} else {
+		miner.config.MaxDABlockSize = new(big.Int).Set(maxBlockSize)
+	}
+	miner.confMu.Unlock()
+}
+
```

Comment:
```suggestion
// SetMaxDASize sets the maximum data availability tx size currently allowed for inclusion. 0 or nil means no maximum.
func (miner *Miner) SetMaxDATxSize(maxTxSize *big.Int) {
	miner.confMu.Lock()
	if maxTxSize != nil && maxTxSize.BitLen() == 0 {
		maxTxSize = nil
	}
	miner.config.MaxDATxSize = maxTxSize
	miner.confMu.Unlock()
}

// SetMaxDABlockSize sets the maximum data availability block size currently allowed for inclusion. 0 or nil means no maximum.
func (miner *Miner) SetMaxDABlockSize(maxBlockSize *big.Int) {
	miner.confMu.Lock()
	if maxBlockSize != nil && maxBlockSize.BitLen() == 0 {
		maxBlockSize = nil
	}
	miner.config.MaxDABlockSize = maxBlockSize
	miner.confMu.Unlock()
}
```

Reply:
I debated this... thought it would be better to have a single method to reduce the # of RPC calls the automated control system would have to do.  But also don't feel strongly!

Reply:
If we assume to usually set both together, it's nice to have the ability to set them atomically within a single RPC.